kong/schema_service.go

```diff
@@ -0,0 +1,29 @@
+package kong
+
+import (
+	"context"
+	"fmt"
+)
+
+// AbstractSchemaService handles schemas in Kong.
+type AbstractSchemaService interface {
+	// Get fetches an entity schema from Kong.
+	Get(ctx context.Context, entity string) (map[string]interface{}, error)
+}
+
+// SchemaService handles schemas in Kong.
+type SchemaService service
+
+// Get retrieves the full schema of kong entities.
+func (s *SchemaService) Get(ctx context.Context, entity string) (map[string]interface{}, error) {
```

Comment:
make a type 
```
type map[string]interface{} Schema
```
return that here and then take this as input to the util package. Might result in import cycles.